### PR TITLE
Update survivIoAim.js

### PR DESCRIPTION
**Pull request by ZXT-Tech**
gameOver now returns false or 0

### CODE REVIEW

File: survivIoAim.js [hunk 1]
```diff
@@ -221,7 +221,7 @@
 
 	var iterate = function() {
 		// check if we in game
-		if(game.gameOver) {
+		if(game.gameOver !== false) {
```
Comment (w3x731) [inline survivIoAim.js]: Why is not
`if(game.gameOver === true) {`

Comment (ZXT-Tech) [reply survivIoAim.js]: Because gameOver does not return true, it returns 0 when it's true.

Comment (w3x731) [reply survivIoAim.js]: Can't see. Which browser do you use? In my chromium console, a `game.gameOver` variable contains only boolean values.

Comment (ZXT-Tech) [reply survivIoAim.js]: Weird, I'm using Chrome.

Comment (w3x731) [reply survivIoAim.js]: default

Comment (w3x731) [reply survivIoAim.js]: Strange. Thanks for reply.

**Pull request merged**